demo/src/app/load-locales.ts

```diff
@@ -0,0 +1,16 @@
+import { defineLocale, LocaleData } from 'ngx-bootstrap/chronos';
```

Comment:
Explain (with docs) why we need that

Reply:
Do the locale review for this component, as discussed in person.

Reply:
this is explained in README.md on section Locales